index.js

```diff
@@ -27,4 +27,18 @@ app.post("/backup", async (req, res) => {
   }
 });
 
+app.delete("/delete", async (req, res) => {
+  try {
+    const daneDoUsuniecia = req.body;
+
+    const result = baseApi.delete("https://httpbin.org/delete", {
+      daneDoUsuniecia,
+    });
+
+    res.status(200);
```

Comment:
To niestety nie ma prawa zadziałać, ponieważ zapytanie nie zakończy się odpowiedzią. Prośba o użycie send np. res.status(200).send('some text');